task_sdk/src/airflow/sdk/__init__.py

```diff
@@ -16,6 +16,32 @@
 # under the License.
 from __future__ import annotations
 
+from typing import TYPE_CHECKING
 
-def hello() -> str:
-    return "Hello from task-sdk!"
+__all__ = ["DAG", "BaseOperator", "TaskGroup"]
+
+if TYPE_CHECKING:
+    from airflow.sdk.definitions.baseoperator import BaseOperator as BaseOperator
+    from airflow.sdk.definitions.dag import DAG as DAG
+    from airflow.sdk.definitions.edges import EdgeModifier as EdgeModifier
+    from airflow.sdk.definitions.taskgroup import TaskGroup as TaskGroup
```

Comment:
Since `as` part contains same name for them
```suggestion
    from airflow.sdk.definitions.baseoperator import BaseOperator
    from airflow.sdk.definitions.dag import DAG
    from airflow.sdk.definitions.edges import EdgeModifier
    from airflow.sdk.definitions.taskgroup import TaskGroup
```

Reply:
The `X as X` is how you signal to ruff et al that this import is designed to "re-export" the item. Without this (I think, i'll check) the import tidy will remove them